radix-transaction-scenarios/src/scenarios/account_locker.rs

```diff
@@ -66,7 +121,10 @@ impl ScenarioCreator for AccountLockerScenarioCreator {
                                     },
                                 )
                                 .try_deposit_entire_worktop_or_abort(
-                                    config.account_locker_admin_account.address,
+                                    state
+                                        .account_locker_admin_account
+                                        .get()
```

Comment:
You can just use `unwrap()`

Reply:
Ah nice! I've switched all places to use `State::unwrap` instead of `State::get`